miniapps/nurbs/nurbs_patch_ex1.cpp

```diff
@@ -0,0 +1,340 @@
+//               MFEM Example 1 - NURBS with patch-wise assembly
+//
+// Compile with: make nurbs_patch_ex1
+//
+// Sample runs:  nurbs_patch_ex1 -incdeg 3 -ref 2 -iro 8 -patcha
+//               nurbs_patch_ex1 -incdeg 3 -ref 2 -iro 8 -patcha -pa
+//               nurbs_patch_ex1 -incdeg 3 -ref 2 -iro 8 -patcha -fint
```

Comment:
All three sample runs crash my GLVis with the same error:
```NURBSFECollection: unknown geometry type.```

I just noticed `nurbs_ex1` also produces this error so this probably is not caused by this PR.

Reply:
It looks like this error was caused by something I changed in PR #3349 a couple months ago. I'll put together a small bugfix PR to correct it.

Reply:
See PR #3447 for a potential fix.